run.js

```diff
@@ -123,6 +138,15 @@ function getMlbTeams() {
     REQUEST(options)
       .then(function (response) {
         mlbTeams = JSON.parse(response);
+
+        if (saveData) {
```

Comment:
where's `saveData` getting modified? i only see it being set on L27.

Reply:
Yeah, it's kinda a one shot deal to write out the current data to files. I didn't want to delete in case I needed to do it again. I will document it better.

Reply:
Actually, I think it best to delete this and put it into a separate script that can be run outside of the main code. So hold off on any more reviews.